src/wdio/services/ServeStaticService.js

```diff
@@ -2,19 +2,74 @@ import serveStatic from '../../../scripts/serve/serve-static';
 import SERVICE_DEFAULTS from '../../../config/wdio/services.default-config';
 import Logger from '../../../scripts/utils/logger';
 
+const WebpackDevServer = require('webpack-dev-server');
+const webpack = require('webpack');
+const ip = require('ip');
+
 const { serveStatic: SERVE_STATIC_DEFAULTS } = SERVICE_DEFAULTS;
 const context = '[Terra-Toolkit:serve-static-service]';
 
+const displayServer = (localAddress, networkAddress) => {
+  Logger.log('Server started listening at', { context });
+  Logger.log(`* Local:            ${Logger.emphasis(localAddress)}`);
+  Logger.log(`* On your network:  ${Logger.emphasis(networkAddress)}`);
+};
+
+// Create a webpack dev server instance.
+const wds = (options) => {
+  const {
+    port, index, locale,
+  } = options;
+  const host = '0.0.0.0';
+  let { config } = options;
+  // if config is a function, execute it with prod mode if applicable.
+  if (typeof config === 'function') {
+    const env = { ...locale && { defaultLocale: locale } };
+    config = config(env, { p: true });
+  }
+
+  // pull the dev server options out of the webpack config. override host, port, and stats. SRY.
+  const devServerOptions = Object.assign({}, config.devServer, {
+    // Disable hot reloading
+    hot: false,
+    inline: false,
+    host,
+    port,
+    index,
+    stats: {
+      colors: true,
+      children: false,
+    },
+  });
+
+  // get a compiler
+  const compiler = webpack(config);
+  // get a server
+  const devServer = new WebpackDevServer(compiler, devServerOptions);
+
+  // start that server.
+  return new Promise((resolve, reject) => {
+    devServer.listen(port, host, (err) => {
+      if (err) {
+        reject(err);
+      }
+      const localAddress = `http://${host}:${port}/`;
+      const networkAddress = `http://${ip.address()}:${port}/`;
+      displayServer(localAddress, networkAddress);
+      resolve(devServer);
+    });
+  });
+};
+
 export default class ServeStaticService {
   async onPrepare(config = {}) {
-    const { site, webpackConfig, locale } = config;
+    const { site, webpackConfig, locale = process.env.LOCALE } = config;
```

Comment:
I would have expected locale to be set by the process.env in the wdio.config, not in the service

Reply:
Actually, [below I had commented](https://github.com/cerner/terra-toolkit/pull/268/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md&file-filters%5B%5D=.png&hide-deleted-files=true#diff-c6cf2bceff376d6c36d8c70fc6db3bb1R75)
> // Explicitly not providing a fallback locale. Providing a fallback will lock the locale for all test runs when using the tt-wdio-runner.

Does setting this value here allow for local changing?

Reply:
I don't think i needed to do this, the wdio config already defaults locale.

Reply:
I lied, i'm going to keep it this way because it works. I think the env gets set through the test run so we have to keep reading it.